ci/axis/rapidsai-driver.yml

```diff
@@ -16,9 +16,7 @@ RAPIDS_VER:
 CUDA_VER:
   - 11.5
   - 11.4
-  - 11.3
```

Comment:
same comments about the driver versions for this file. let's remove all 455 and 465 mentions.